packages/service-library/tests/aiohttp/long_running_tasks/test_long_running_tasks_with_task_context.py

```diff
@@ -169,19 +176,35 @@ async def test_cancel_task(
 ):
     assert client_with_task_context.app
     task_id = await start_long_running_task(client_with_task_context)
-    cancel_url = client_with_task_context.app.router["cancel_and_delete_task"].url_for(
+    cancel_url = client_with_task_context.app.router["remove_task"].url_for(
         task_id=task_id
     )
     # calling cancel without task context should find nothing
-    resp = await client_with_task_context.delete(f"{cancel_url}")
-    await assert_status(resp, status.HTTP_404_NOT_FOUND)
+    # no longer waits for removal to end
+    async for attempt in AsyncRetrying(
+        wait=wait_fixed(0.1),
+        stop=stop_after_delay(5),
+        reraise=True,
+        retry=retry_if_exception_type(AssertionError),
+    ):
+        with attempt:
+            resp = await client_with_task_context.delete(f"{cancel_url}")
+            await assert_status(resp, status.HTTP_404_NOT_FOUND)
     # calling with context should find and delete the task
```

Comment:
this is a weird thing to do after the comment in line 182.
you want to check that the task is not around for 5 seconds right? now you probably do not check anything with this.